packages/node_modules/@webex/plugin-meetings/src/locus-Info/index.js

```diff
@@ -0,0 +1,360 @@
+import isEqual from 'lodash.isequal';
+
+import {LOCUSEVENT} from '../constants';
+import Parser from '../locus-Info';
+import InfoUtils from '../locus-Info/utils/info';
+import selfUtils from '../locus-Info/utils/self';
+
+/**
+ * @export
+ * @class LocusInfo
+*/
+export default class LocusInfo {
+  constructor(callback) {
+    this.setup({});
+    this.updateMeeting = callback;
+    this.emitChanges = false;
+  }
+
+  /**
+  * updates the default values from locus
+  * @param {object} locus
+  */
+  setup(locus) {
+    this.controls = locus.controls || null;
+    this.conversationUrl = locus.conversationUrl || null;
+    this.created = locus.created || null;
+    this.fullState = locus.fullState || null;
+    this.host = locus.host || null;
+    this.info = locus.info || null;
+    this.mediaShares = locus.mediaShares || null;
+    this.participantsUrl = locus.participantsUrl || null;
+    this.replaces = locus.replaces || null;
+    this.self = locus.self || null;
+    this.url = locus.url || null;
+    this.aclUrl = locus.aclUrl || null;
+    this.baseSequence = locus.baseSequence || null;
+    this.sequence = locus.sequence || null;
+    this.membership = locus.membership || null;
+    this.identities = locus.identities || null;
+    this.participants = locus.participants || null;
+  }
```

Comment:
arun please comment

Reply:
Some time we set the values to null or actual value . Later we add those values from the meeting object so we had to make it separate 

using callback because it was just one place where we required it. Can switch to events if needed later

Reply:
Ok re: the constructor. I don't think we should use a callback. We can discuss as a broader group if you like.